recipes/ccr/start.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -e
+
+source .elastic-version
+
+# Start metrics store
+docker-compose -f ./metricstore-docker-compose.yml up -d 
+
+# Start Elasticsearch
+docker-compose up -d
+
+printf "Waiting for clusters to get ready "
```

Comment:
Is there a reason you're sometimes using `printf` and sometimes `echo`?

Reply:
Yes to avoid newline characters printed (to facilitate printing `...` as progress)